### PR TITLE
document change of -verify behavior

### PR DESCRIPTION
**Pull request by vladak**
Follow up to PR #23773

### CODE REVIEW

File: doc/man1/openssl-crl.pod.in [hunk 1]
```diff
@@ -162,6 +162,10 @@ L<openssl-ca(1)>,
 L<openssl-x509(1)>,
 L<ossl_store-file(7)>
 
+=head1 HISTORY
+
+Since OpenSSL 3.3, the B<-verify> will exit with 1 on failure.
```
Comment (t8m) [inline doc/man1/openssl-crl.pod.in:167]: Nit: Either remove `the` or add `option`, please.

File: doc/man1/openssl-req.pod.in [hunk 2]
```diff
@@ -794,6 +794,8 @@ Since OpenSSL 3.2,
 generated certificates bear X.509 version 3 unless B<-x509v1> is given,
 and key identifier extensions are included by default.
 
+Since OpenSSL 3.3, the B<-verify> will exit with 1 on failure.
```
Comment (t8m) [inline doc/man1/openssl-req.pod.in:797]: Same as above.

**Review by t8m — CHANGES_REQUESTED** [review]

**Review by t8m — APPROVED** [review]

**Comment by vt-alt** [comment]
Btw, for gost-engine tests `openssl crl` was not called with `-verify` but still returns failure.
```
 test -skip {![file exists test.crl]} "Verifying corrupted CRL" {
        makeFile "badcrl.pem" [hackPem "\01\x1E" [getFile test.crl] "\01\0"]
        grep verify [openssl "crl -in badcrl.pem -noout -CAfile $::test::ca/cacert.pem"]
 } -1 "verify failure"
```

**Review by tom-cosgrove-arm — APPROVED** [review]

**Comment by vt-alt** [comment]
> Btw, for gost-engine tests `openssl crl` was not called with `-verify` but still returns failure.

It's not stated that `-verify` option is default and `crl` exits with failure even without `-verify` option so the proposed text is not entirely correct. Or there will be third PR to correct this?

**Comment by t8m** [comment]
Merged to the master and 3.3 branches. Thank you for your contribution.

**Comment by vt-alt** [comment]
Thanks for the reply. (But why someone should send another PR and not the original PR author correct theirs? Like in any normal review cycle. Of course who am I to criticize, but these two last PRs just make me curious of OTC review process.)

**Comment by t8m** [comment]
@vt-alt The fact that in openssl crl `-verify` is implied by some other options is not really related to this PR. It also belongs to all branches.

I've opened: https://github.com/openssl/openssl/pull/24024

**Pull request closed without merging**